src/debug_menu.cpp

```diff
@@ -4467,7 +4467,6 @@ void debug()
                     _( "Quit without saving?  This may cause issues such as duplicated or missing items and vehicles!" ) ) ) {
                 player_character.set_moves( 0 );
                 g->uquit = QUIT_NOSAVED;
-                g->save_is_dirty = true;
```

Comment:
You can make something like
```c++
            if( !g->save_is_dirty && query_yn( "Mark the save file as dirty? It will disable saving it, preventing any accidental modifications in the future." ) ) {
				g->save_is_dirty = true;
}
```
but i presume it will be annoying to say "no" every single time you reload the save

Reply:
I don’t mind that as long as my workflow can exist.